pallets/nfts/src/tests.rs

```diff
@@ -2891,6 +3449,8 @@ fn claim_swap_should_work() {
 			default_item_config(),
 		));
 
+		assert_eq!(AccountBalance::<Test>::get(collection_id, user_1.clone()), 2);
+		assert_eq!(AccountBalance::<Test>::get(collection_id, user_2.clone()), 3);
```

Comment:
If these are for the above mints they can be removed as they are tested in the mint function